tests/tics-scan.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(dirname "$BASH_SOURCE")
```

Comment:
I tend to favor this, because it resolves to the script's absolute path:

```suggestion
SCRIPT_DIR=$(realpath $(dirname "$BASH_SOURCE"))
```